### PR TITLE
Gardening.

### LINKED ISSUE #8: Add Proper .gitignore

### PR DESCRIPTION
**Pull request by andersio**
Pull in `.gitignore` from our own mainline repo.
Break function signatures that go beyond 90 characters.
Tweaked the argument labels of `SignalProducer.system`.

fixes #8

### CODE REVIEW

File: ReactiveFeedback/Sources/FeedbackLoops.swift [hunk 1]
```diff
@@ -3,45 +3,56 @@ import ReactiveSwift
 import enum Result.NoError
 
 public struct FeedbackLoop<State, Event> {
-    let loop: (Scheduler, Signal<State, NoError>) -> Signal<Event, NoError>
+    public let loop: (Scheduler, Signal<State, NoError>) -> Signal<Event, NoError>
```
Comment (sergdort) [inline ReactiveFeedback/Sources/FeedbackLoops.swift:6]: I think this should not be `public`

File: ReactiveFeedback/Sources/SignalProducer+System.swift [hunk 2]
```diff
@@ -2,29 +2,33 @@ import Foundation
...
-                                     reduce: @escaping Reducer<Value, Event>,
-                                     feedback: [FeedbackLoop<Value, Event>]) -> SignalProducer<Value, NoError> {
-        return SignalProducer.deferred {
-            let (subject, observer) = Signal<Value, NoError>.pipe()
-            let events = Signal<Event, NoError>.merge(feedback.map { feedback in
-                return feedback.loop(scheduler, subject)
-            })
-            return SignalProducer(events.scan(initialState, reduce))
-                .prefix(value: initialState)
+            return SignalProducer<Event, NoError>(Signal.merge(events))
```
Comment (sergdort) [inline ReactiveFeedback/Sources/SignalProducer+System.swift:19]: Why remove `deferred`?

Comment (sergdort) [reply ReactiveFeedback/Sources/SignalProducer+System.swift:19]: Oh, my bad did not notice it =)

**Review by sergdort — CHANGES_REQUESTED** [review]
👍 

I dont think we should expose `loop` publicly

File: ReactiveFeedback/Sources/SignalProducer+System.swift (hunk 2 above)
Comment (RuiAAPeres) [reply ReactiveFeedback/Sources/SignalProducer+System.swift:19]: yet to understand the usage of `deferred`, based on the discussions we had on Slack. ☹️

**Comment by andersio** [comment]
@sergdort For testing? If it isn't exposed, I am not sure how API consumers can test it intuitively.

**Review by sergdort — APPROVED** [review]

**Pull request merged**